crates/ln-dlc-node/src/node.rs

```diff
@@ -1056,8 +1056,8 @@ impl Node {
         // The contract input to be used for setting up the trade between the trader and the
         // coordinator
         let contract_input = ContractInput {
-            offer_collateral: margin_coordinator,
-            accept_collateral: margin_trader,
+            offer_collateral: margin_trader,
+            accept_collateral: margin_coordinator,
```

Comment:
🤔 I'm confused about this `trade` API being in this crate in the first place. I think this should be defined by the app crate (if such a thing exists).

Reply:
To clarify: You mean this should not be in the `ln-dlc-node` but it should be in the `coordinator`, correct? If that's what you mean then I agree :)